rpms/generate_rpm_package.sh

```diff
@@ -93,7 +94,7 @@ build_rpm() {
         ${CONTAINER_NAME} ${TARGET} ${BRANCH} ${ARCHITECTURE} \
         ${JOBS} ${REVISION} ${INSTALLATION_PATH} ${DEBUG} \
         ${CHECKSUM} ${PACKAGES_BRANCH} ${USE_LOCAL_SPECS} ${SRC} \
-        ${LEGACY} ${LOCAL_SOURCE_CODE} || return 1
+        ${LEGACY} "${LOCAL_SOURCE_CODE}" "${DATABASE_OUTPUT}" || return 1
```

Comment:
As you did here. Check the last comment in the Debian script and name the variables equally in both scripts. 

PS: you don't need to quote the variables if they don't contain any spaces.